plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -341,9 +341,9 @@ def move_to_target_dir(download_path) -> str:
                     error_msg = f"Expected a single directory, found: {paths}"
                     raise DownloadError(error_msg)
                 package_path = paths.pop()
-
-            package_path.rename(target_dir / package_path.name)
-            return str(package_path)
+            # move not rename cause can be on different filesystems
+            shutil.move(package_path, target_dir / package_path.name)
+            return str(target_dir / package_path.name)
```

Comment:
@solarw can you explain the issue with [Path.rename](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rename)? An example would be nice.